server/src/main/java/org/eclipse/hono/server/HonoServer.java

```diff
@@ -302,7 +304,7 @@ protected int getPortDefaultValue() {
      *
      * @return the port Hono listens on.
      */
-    public int getHonoServerPort() {
+    public int getActualPort() {
```

Comment:
As long as we never want this functionality be overwritten in a subclass, I agree. Wasn't 100% sure about it here, so left it as it was (no final). Will change it.

Will improve JavaDoc, but mainly it was correct I think (?). Maybe discuss again after next update...